.github/workflows/mark-issue-stale.yml

```diff
@@ -13,12 +14,14 @@ jobs:
           repo-token: ${{ secrets.GITHUB_TOKEN }}
           stale-issue-message: 'This issue is stale because it has been 180 days with no activity. You can keep the issue open by adding a comment. If you do, please provide additional context and explain why you’d like it to remain open. You can also close the issue yourself — if you do, please add a brief explanation and apply one of relevant issue close labels.'
           # Days before issue is considered stale.
-          days-before-stale: 180
+          days-before-issue-stale: 180
           # Exempt issue labels.
           exempt-issue-labels: '[Pri] High,[Pri] BLOCKER,[Status] Keep Open'
           # Disable auto-close of both issues and PRs.
           days-before-close: -1
           # Get issues in ascending (oldest first) order.
-          ascending: true
+          ascending: false
```

Comment:
Isn't it faster to get older ones first?

Reply:
I was hoping to address a behavior issue I noticed with this change, but I'm not sure if it does.

This stale bot keeps going through all issues in ascending or descending order. In the week or so it's been running, it's tagged 200+ issues with stale. I then noticed the numbers have not gone up since a few days ago.

It turns out the stale bot is going through the same set of issues each time. These are issues that survived the initial cull and so have reasons not to have the stale label applied to it.

To compare, check out [this run](https://github.com/Automattic/wp-calypso/runs/1948383016?check_suite_focus=true) and [this run](https://github.com/Automattic/wp-calypso/runs/1944143864?check_suite_focus=true) and notice how it's the exact same set of issues being processed.

I will investigate if I can tweak other levers to make it do what we want.

Reply:
I've posted a question to the maintainers of stale bot, but in the meantime I will increase the `operations-per-run` figure to 2000 - enough to cover the number of would-be stale issues in our repo, and still under the 15,000 limit on API interactions that GitHub imposes on enterprise customers.